icekit/workflow/models.py

```diff
@@ -69,6 +69,14 @@ def __str__(self):
 
 class WorkflowStateMixin(models.Model):
     workflow_states = GenericRelation(WorkflowState)
+    brief = models.TextField(
+        blank=True,
+        help_text="A document brief describing the purpose of this item"
+    )
+    admin_notes = models.TextField(
```

Comment:
@cogat I think _glamkit-collections_ might be the only ICEkit/GLAMkit project that is *not* affected by the changes to `WorkflowStateMixin`